### PR TITLE
Create evidence-based-complementary-and-alternative-medicine.csl

### PR DESCRIPTION
**Pull request by KMDjustin**
This is a new csl style for the medicine journal, eCAM (http://www.hindawi.com/journals/ecam/).

### CODE REVIEW

File: evidence-based-complementary-and-alternative-medicine.csl [hunk 1]
```diff
@@ -0,0 +1,313 @@
...
+    <id>http://www.zotero.org/styles/evidence-based-complementary-and-alternative-medicine</id>
+    <link href="http://www.zotero.org/styles/evidence-based-complementary-and-alternative-medicine" rel="self"/>
+    <link href="http://www.hindawi.com/journals/ecam/guidelines/" rel="documentation"/>
+    <author>
+      <name>Seungwon Shin</name>
+      <email>justin0624@gmail.com</email>
+    </author>
+    <category citation-format="numeric"/>
+    <category field=" medicine"/>
+    <category field=" biology"/>
```
Comment (rmzelle) [inline evidence-based-complementary-and-alternative-medicine.csl]: Can you remove the spaces here? " medicine" and " biology" should be "medicine" and "biology", respectively.

File: evidence-based-complementary-and-alternative-medicine.csl [hunk 2]
```diff
@@ -0,0 +1,313 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="sort-only">
```
Comment (rmzelle) [inline evidence-based-complementary-and-alternative-medicine.csl]: Please see https://github.com/citation-style-language/styles/wiki/Style-Requirements#3---style-locale. If this journal is English-only, please set the "default-locale" to British or US English.

File: evidence-based-complementary-and-alternative-medicine.csl [hunk 3]
```diff
@@ -0,0 +1,313 @@
...
+    <link href="http://www.hindawi.com/journals/ecam/guidelines/" rel="documentation"/>
+    <author>
+      <name>Seungwon Shin</name>
+      <email>justin0624@gmail.com</email>
+    </author>
+    <category citation-format="numeric"/>
+    <category field=" medicine"/>
+    <category field=" biology"/>
+    <eissn>1741-4288</eissn>
+    <issnl>1741-427X</issnl>
```
Comment (rmzelle) [inline evidence-based-complementary-and-alternative-medicine.csl]: Please replace `<issnl>1741-427X</issnl>` by `<issn>1741-427X</issn>`

File: evidence-based-complementary-and-alternative-medicine.csl [hunk 4]
```diff
@@ -0,0 +1,313 @@
...
+        <if locator="page">
+          <label variable="locator" form="short"/>
+        </if>
+        <else>
+          <label variable="locator" form="short" text-case="capitalize-first"/>
+        </else>
+      </choose>
+      <text variable="locator"/>
+    </group>
+  </macro>
```
Comment (rmzelle) [inline evidence-based-complementary-and-alternative-medicine.csl]: Please remove the entire "citation-locator" macro above. The macro is not used in the style, and therefore should be removed.

**Comment by rmzelle** [comment]
Great, thanks. We can accept the style once you update it with the small requests I made.

**Comment by KMDjustin** [comment]
Thanks for your help.

I corrected what you mentioned.

Please let me know if there is more.


Thx again

Seungwon Shin



2016-07-05 9:53 GMT+09:00 Rintze M. Zelle <notifications@github.com>:

File: evidence-based-complementary-and-alternative-medicine.csl [hunk 5]
```diff
@@ -0,0 +1,299 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" default-locale="US English" demote-non-dropping-particle="sort-only">
```
Comment (rmzelle) [inline evidence-based-complementary-and-alternative-medicine.csl]: @KMDjustin, can you change `default-locale="US English"` to `default-locale="en-US"`? (it has to be a proper locale code)

File: evidence-based-complementary-and-alternative-medicine.csl [hunk 6]
```diff
@@ -0,0 +1,299 @@
...
+        </choose>
+      </if>
+    </choose>
+  </macro>
+  <macro name="page">
+    <group>
+      <label variable="page" form="short" suffix=" "/>
+      <text variable="page"/>
+    </group>
+  <citation collapse="citation-number">
```
Comment (rmzelle) [inline evidence-based-complementary-and-alternative-medicine.csl]: @KMDjustin, can you change

```xml
    </group>
  <citation collapse="citation-number">
```

to

```xml
    </group>
  </macro>
  <citation collapse="citation-number">
```

? You accidentally deleted one ` </macro>` too many.

**Comment by KMDjustin** [comment]
I have changed the lines that you pointed out. 

Thx for your support again~!

**Comment by rmzelle** [comment]
Thanks!

**Pull request merged**